presto-spi/src/main/java/io/prestosql/spi/connector/TopNFunction.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.spi.connector;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class TopNFunction
```

Comment:
I am not sure this is a "function".
I will defer to Martin for the name though

Reply:
renamed it to just TopN